torchvision/ops/boxes.py

```diff
@@ -167,25 +167,35 @@ def box_convert(boxes: Tensor, in_fmt: str, out_fmt: str) -> Tensor:
             boxes_xyxy = _box_xywh_to_xyxy(boxes)
             if out_fmt == "cxcywh":
                 boxes_converted = _box_xyxy_to_cxcywh(boxes_xyxy)
+            else:
+                raise ValueError("Unsupported Boundig Box Conversions for given in_fmt and out_fmt")
```

Comment:
It would be nice to add tests to improve the coverage.

Reply:
There is assert statement before these `ValueError` codeblocks. So the test will never reach this stage. Should I remove the `assert` statements?

Reply:
yes, please

Reply:
Added 👍